[BUG] SIGSEGV if cloned VM disk is in the different storage

**Describe the bug**
Plugin throws a SIGSEGV error if cloned VM disk is in the different storage.

```
╷
│ Error: Plugin did not respond
│ 
│   with module.ubuntu_vm.proxmox_virtual_environment_vm.ubuntu_vm,
│   on ../terraform-proxmox-ubuntu2004/main.tf line 1, in resource "proxmox_virtual_environment_vm" "ubuntu_vm":
│    1: resource "proxmox_virtual_environment_vm" "ubuntu_vm" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-proxmox_v0.4.6_x4 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xd87a19]

goroutine 22 [running]:
github.com/danitso/terraform-provider-proxmox/proxmox.(*VirtualEnvironmentClient).MoveVMDisk(0xc000678460, 0xc00082c280, 0x7, 0x67, 0xc0004c6500, 0x708, 0xc000535860, 0x2)
        /home/pasha/code/terraform-provider-proxmox/proxmox/virtual_environment_vm.go:164 +0x79
github.com/danitso/terraform-provider-proxmox/proxmoxtf.resourceVirtualEnvironmentVMCreateClone(0xc0003eb730, 0xf71380, 0xc000678460, 0xee7b40, 0xc000841f38)
        /home/pasha/code/terraform-provider-proxmox/proxmoxtf/resource_virtual_environment_vm.go:1480 +0x2808
github.com/danitso/terraform-provider-proxmox/proxmoxtf.resourceVirtualEnvironmentVMCreate(0xc0003eb730, 0xf71380, 0xc000678460, 0x2, 0x182d760)
        /home/pasha/code/terraform-provider-proxmox/proxmoxtf/resource_virtual_environment_vm.go:1056 +0x88
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc0000be870, 0xc00047e140, 0xc0005f0d40, 0xf71380, 0xc000678460, 0xf43701, 0xc00016bd18, 0xc000390b10)
        /home/pasha/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.16.0/helper/schema/resource.go:310 +0x375
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc0003a0280, 0xc00059fa38, 0xc00047e140, 0xc0005f0d40, 0xc00030c088, 0xc00012cbf0, 0xf457c0)
        /home/pasha/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.16.0/helper/schema/provider.go:294 +0x99
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc000010f58, 0x1253fd0, 0xc00024c1b0, 0xc00026a000, 0xc000010f58, 0xc00024c1b0, 0xc000646ba0)
        /home/pasha/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.16.0/internal/helper/plugin/grpc_provider.go:885 +0x8a5
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x1039780, 0xc000010f58, 0x1253fd0, 0xc00024c1b0, 0xc000264060, 0x0, 0x1253fd0, 0xc00024c1b0, 0xc000269000, 0x7a2)
        /home/pasha/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.16.0/internal/tfplugin5/tfplugin5.pb.go:3305 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000311dc0, 0x125d398, 0xc000582600, 0xc000266000, 0xc0004974d0, 0x17edaa0, 0x0, 0x0, 0x0)
        /home/pasha/go/pkg/mod/google.golang.org/grpc@v1.34.0/server.go:1210 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc000311dc0, 0x125d398, 0xc000582600, 0xc000266000, 0x0)
        /home/pasha/go/pkg/mod/google.golang.org/grpc@v1.34.0/server.go:1533 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc00003e3a0, 0xc000311dc0, 0x125d398, 0xc000582600, 0xc000266000)
        /home/pasha/go/pkg/mod/google.golang.org/grpc@v1.34.0/server.go:871 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /home/pasha/go/pkg/mod/google.golang.org/grpc@v1.34.0/server.go:869 +0x1fd

Error: The terraform-provider-proxmox_v0.4.6_x4 plugin crashed!
```

--- a/proxmox/virtual_environment_vm.go
+++ b/proxmox/virtual_environment_vm.go
@@ -161,12 +161,12 @@ func (c *VirtualEnvironmentClient) GetVMStatus(nodeName string, vmID int) (*Virt
 func (c *VirtualEnvironmentClient) MoveVMDisk(nodeName string, vmID int, d *VirtualEnvironmentVMMoveDiskRequestBody, timeout int) error {
 	taskID, err := c.MoveVMDiskAsync(nodeName, vmID, d)
 
-	if strings.Contains(err.Error(), "you can't move to the same storage with same format") {
-		// if someone tries to move to the same storage, the move is considered to be successful
-		return nil
-	}
-
 	if err != nil {
+		if strings.Contains(err.Error(), "you can't move to the same storage with same format") {
+			// if someone tries to move to the same storage, the move is considered to be successful
+			return nil
+		}
+
 		return err
 	}
 

